src/niclips/image/_convert.py

```diff
@@ -102,4 +109,7 @@ def to_iso_ras(img: nib.nifti1.Nifti1Image) -> nib.nifti1.Nifti1Image:
             img = resample_img(
                 img, target_affine=target_affine, interpolation="nearest"
             )
+    # Set filepath to original incase it is needed
+    if img_path:  # pragma: no cover
```

Comment:
It's never stricly needed AFAIK in the existing anatomical / functional images. It is needed for diffusion since we don't track .bval and .bvec files. I'm open to better ways to keeping track of non-imaging files (that are related to the image) here. There might be other situations where this could be useful.

> Also # pragma: no cover is cheating :P 

🙈